src/environment.js

```diff
@@ -23,6 +23,11 @@ const ENV_VARS = {
     '<username>/<repo>/contents/<path_to_addressbook.json>/addressbook.json',
     true,
   ],
+  GITHUB_LEDGER_FILE_PATH: [
+    process.env.GITHUB_LEDGER_FILE_PATH,
+    '<account>/<repo>/<branch>/<path_to_file>/accounts.json',
```

Comment:
This file is not the ledger, it the accounts file. The variable should be named "GITHUB_ACCOUNTS_FILE_PATH"

Reply:
Name changed to GITHUB_ACCOUNTS_FILE_PATH